apps/bot/src/bot/handlers/trending.ts

```diff
@@ -0,0 +1,113 @@
+import { Context } from "telegraf";
+import { FastifyInstance } from "fastify";
+import { trendingService } from "../../services/trending.service";
+import { TRENDING_MESSAGES } from "../../constants/trending.constants";
+import { getTrendingKeyboard } from "../keyboards/trending-menu";
+
+interface BotContext extends Context {
+  userId?: string;
+}
+
+export async function trendingHandler(
+  ctx: BotContext,
+  _server: FastifyInstance
+) {
+  try {
+    const loading = await ctx.reply(TRENDING_MESSAGES.FETCHING);
+    const chatId = ctx.chat!.id;
+
+    await trendingService.loadApiPage(chatId, 0);
+
+    const st = trendingService.getState(chatId)!;
+    const text = trendingService.formatPage(st.items, st.page);
+    const kb = getTrendingKeyboard(chatId);
+
+    const sent = await ctx.reply(text, {
+      parse_mode: "Markdown",
+      link_preview_options: { is_disabled: true } as any,
+      reply_markup: kb.reply_markup,
+    } as any);
+
+    trendingService.setMessageId(chatId, sent.message_id);
+
+    try {
+      await ctx.telegram.deleteMessage(chatId, loading.message_id);
+    } catch {}
```

Comment:
```suggestion
      await ctx.telegram.deleteMessage(chatId, loading.message_id);
```
Không cần thiết phải bỏ trong try/catch như vậy.